### PR TITLE
Add (basic) BOUNTIES.md

### PR DESCRIPTION
**Pull request by jacogr**

### CODE REVIEW

File: BOUNTIES.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Bounties
+
+From time-to-time we will add bounties for features.
+
+These are generously provided by the [Web3 Foundation](https://web3.foundation/) and as such any employees of Parity or those of the W3F are eligible for them. (This includes people being being by either party for development or community work, related or un-related to polkadot-js).
```
Comment (ltfschoen) [inline BOUNTIES.md]: `eligible` should be `ineligible`

File: BOUNTIES.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+These are generously provided by the [Web3 Foundation](https://web3.foundation/) and as such any employees of Parity or those of the W3F are ineligible for them. (This includes people being being by either party for development or community work, related or un-related to polkadot-js).
+
+The idea is that these bounties should be left open to community participation, so only if there is no outside interest for a specific issue, should those directly or indirectly paid by the W3F for work, attempt to close an issue. (in which case it will be "un-bountied")
+
+Any current bounties are tracked by the [!bounty](https://github.com/polkadot-js/apps/labels/%21bounty) label.
+
+## Process
+
+Once listed, the normal [Gitcoin](https://gitcoin.co/) process kicks in. This means application, work and payment is manged by this tool. The values for bounties are determined by the size estimation done by the team.
```
Comment (Tbaut) [inline BOUNTIES.md]: typo
```suggestion
Once listed, the normal [Gitcoin](https://gitcoin.co/) process kicks in. This means application, work and payment is managed by this tool. The values for bounties are determined by the size estimation done by the team.
```

File: BOUNTIES.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Bounties
+
+From time-to-time we will add bounties for features.
+
+These are generously provided by the [Web3 Foundation](https://web3.foundation/) and as such any employees of Parity or those of the W3F are ineligible for them. (This includes people being being by either party for development or community work, related or un-related to polkadot-js).
+
+The idea is that these bounties should be left open to community participation, so only if there is no outside interest for a specific issue, should those directly or indirectly paid by the W3F for work, attempt to close an issue. (in which case it will be "un-bountied")
+
+Any current bounties are tracked by the [!bounty](https://github.com/polkadot-js/apps/labels/%21bounty) label.
```
Comment (Tbaut) [inline BOUNTIES.md]:
```suggestion
Current bounties are tracked by the [!bounty](https://github.com/polkadot-js/apps/labels/%21bounty) label.
```

File: BOUNTIES.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Bounties
+
+From time-to-time we will add bounties for features.
+
+These are generously provided by the [Web3 Foundation](https://web3.foundation/) and as such any employees of Parity or those of the W3F are ineligible for them. (This includes people being being by either party for development or community work, related or un-related to polkadot-js).
```
Comment (Tbaut) [inline BOUNTIES.md]:
```suggestion
These are generously provided by the [Web3 Foundation](https://web3.foundation/) and as such employees of Parity or those of the W3F are ineligible for them. (This includes people being by either Party for development or community work, related or un-related to polkadot-js).
```

**Pull request merged**